### ISSUE TITLE
Translate this project

### ISSUE THREAD

**Issue by luzpaz** [OpenCollaborationPlatform/CollaborativeFC#6]
At some point a translation infrastructure should be set up, right?

**Comment by ickby** [comment]
Yes, definitely. Luckily only little text is required in the UI. Unfortunately I do not yet know how to do that for add-ons. Do you have a link to read up on this?

**Comment by luzpaz** [comment]
https://github.com/FreeCAD/FreeCAD-addons/issues/187
